sherpa/astro/ui/tests/test_filtering.py

```diff
@@ -171,20 +221,29 @@ def test_filter_bad_ungrouped(make_data_path, clean_astro_ui):
     # anything. See issue #1169
     #
     ui.notice(0.5, 7)
+    assert len(caplog.records) == 2
+    clc_filter(caplog, "dataset 1: 0.0073:14.5416 Energy (keV) (unchanged)")
+
     assert pha.mask == pytest.approx(expected)
 
     # We need to ignore to change the mask.
     #
     ui.ignore(None, 0.5)
+    assert len(caplog.records) == 3
+    clc_filter(caplog, "dataset 1: 0.0073:14.5416 -> 0.511:14.5416 Energy (keV)")
+
     ui.ignore(7, None)
+    assert len(caplog.records) == 4
+    clc_filter(caplog, "dataset 1: 0.511:14.5416 -> 0.511:6.9934 Energy (keV)")
```

Comment:
Should at least one of those tests use wavelength instead of energy?

Reply:
At some level I'd hope that the existing notice/filter/... code should check this,as all we're really doing is repackaging existing code, but you are right that we should add a test. So I've done so.